peps/pep-0705.rst

```diff
@@ -363,6 +365,35 @@ Notes:
     b: B = { "x": 3 }
     c: C = a | b  # Accepted by type checker: "x" will always come from b
 
+Minimum support
+^^^^^^^^^^^^^^^
+
+A type checker may reject sound assignments, for instance if it is unable to support bidirectional type inference. This section provides a minimum standard of support that a type checker should provide, so users can write code that will be consistently accepted across all type checkers.
+
+Type checkers may reject any code not conforming to this model, but the error should be explicit that this is a type checker limitation, and that the user code is not necessarily unsound. It may be reasonable to use a distinct code for suppressing such issues, so if the checker later supports full inference and the code is indeed unsound, the issue will no longer be suppressed.
```

Comment:
I don't think you should go into advice about error messages or "distinct codes". That's getting too deep into type checker behaviors and isn't something a PEP should get into.